src/python/grpcio_observability/grpc_observability/python_census_context.h

```diff
@@ -105,6 +105,7 @@ struct Measurement {
   MetricsName name;
   MeasurementType type;
   MeasurementValue value;
+  bool registered_method;
```

Comment:
Should this file be renamed?

Reply:
We might still need this for tracing, I think census sounds general enough, I do need to change `GRPC_PYTHON_OPENCENSUS_H` tho.

Reply:
Discussed offline. We'll go with `python_o11y_context` or `python_observability_context`.